doc/nrf/libraries/nrf_rpc/nrf_rpc_uart.rst

```diff
@@ -10,29 +10,43 @@ nRF RPC UART transport
 The nRF RPC UART transport allows you to use the :ref:`nrf_rpc` library to execute procedures on a remote processor that is connected with the local processor using the UART interface.
 This transport is used in the :ref:`nrf_rpc_ps_samples`.
 
+Configuration
+*************
+
+Set :kconfig:option:`CONFIG_NRF_RPC_UART_TRANSPORT` Kconfig option to enable the nRF RPC UART transport.
+
+Define the ``nordic,rpc-uart`` property in a devicetree overlay file to select the UART peripheral used by the API serialization libraries within |NCS|, such as :ref:`ble_rpc` or :ref:`nfc_rpc`.
```

Comment:
There's two ways to break this sentence into two for readability:
```suggestion
Define the ``nordic,rpc-uart`` property in a devicetree overlay file.
The property is used to select the UART peripheral that is used by the API serialization libraries within the |NCS|, such as :ref:`ble_rpc` or :ref:`nfc_rpc`.
```
or
```suggestion
Define the ``nordic,rpc-uart`` property in a devicetree overlay file to select the UART peripheral.
Select the peripheral used by the API serialization libraries within the |NCS|, such as :ref:`ble_rpc` or :ref:`nfc_rpc`.
```
I think the first option is the correct one, but not sure.